### ISSUE TITLE
Report-width parameter is not working?

### ISSUE THREAD

**Issue by danger89** [squizlabs/PHP_CodeSniffer#3147]
**Describe the bug**
When using `--report-width` parameter, nothing changed (keeps at 120 chars).

**Code sample**
```sh
phpcs --report-width=400 --standard=PSR12 .
```

**To reproduce**
See code example above.

I still get:

```log
----------------------------------------------------------------------
  26 | WARNING | Line exceeds 120 characters; contains 125 characters
  36 | WARNING | Line exceeds 120 characters; contains 146 characters
  41 | WARNING | Line exceeds 120 characters; contains 122 characters
  43 | WARNING | Line exceeds 120 characters; contains 179 characters
 108 | WARNING | Line exceeds 120 characters; contains 205 characters
 122 | WARNING | Line exceeds 120 characters; contains 194 characters
----------------------------------------------------------------------
```

**Expected behavior**

I expect that I do not see warnings anymore regarding lines that are below the given number of characters per line (eg. 400).

**Versions (please complete the following information):**
 - OS: Linux Mint 20
 - PHP: 7.4.3 (CLI)
 - PHPCS: 3.5.8
 - Standard:  PSR12

Thanks.

Regards,
Melroy van den Berg

**Comment by gsherwood** [comment]
That argument doesn't change the behaviour of the line length sniff.

The report width CLI argument is documented as "How many columns wide screen reports should be printed or set to "auto" to use current screen width, where supported". So it's changing settings on the report you see printed to screen.

If you want to change the line length setting, you need to create a custom ruleset like this:
```xml
<?xml version="1.0"?>
<ruleset name="MyStandard">
    <rule ref="PSR12" />
    <rule ref="Generic.Files.LineLength">
        <properties>
            <property name="lineLimit" value="400" />
            <property name="absoluteLineLimit" value="400" />
        </properties>
    </rule>
</ruleset>
```
Or exclude it entirely:
```xml
<?xml version="1.0"?>
<ruleset name="MyStandard">
    <rule ref="PSR12">
        <exclude name="Generic.Files.LineLength" />
    </rule>
</ruleset>
```
Save that to a file called `.phpcs.xml` in your project's root directory and then you can call `phpcs` without using `--standard` and it will use that standard instead. Or you can pass in the path to it directly: `phpcs --standard=/path/to/.phpcs.xml`

More info here: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties

Does that clear things up?

**Comment by danger89** [comment]
Yup, sorry stupid me.

This make so much more sense!
